cartography/intel/azure/sql.py

```diff
@@ -0,0 +1,930 @@
+import logging
```

Comment:
Won't block here, but we should split this file into multiple ones in a `sql/` folder.

Very much appreciate the work but or future PR, let's try to make them smaller if possible :)

Reply:
Alright @achantavy ! How about I make those changes in a future PR? I'll add it to my to-do list :)

Also, updated the branch!